From: heoh <h0h6h2h5@gmail.com>
Subject: [PATCH] WASI: Update static_assert for int64_t max value

---
 zetasql/public/json_value.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zetasql/public/json_value.cc b/zetasql/public/json_value.cc
index 67d324b7..059a1ddc 100644
--- a/zetasql/public/json_value.cc
+++ b/zetasql/public/json_value.cc
@@ -787,7 +787,7 @@ absl::partial_ordering JsonCompareNumber(Int x, double y) {
   static_assert(std::numeric_limits<double>::is_iec559);
   static_assert(std::numeric_limits<double>::digits == 53);
   static_assert(std::numeric_limits<double>::radix == 2);
-#ifdef __EMSCRIPTEN__
+#if defined(__EMSCRIPTEN__) || defined(__wasi__)
   static_assert(std::numeric_limits<int64_t>::max() == (1ULL << 63) - 1);
 #else
   static_assert(std::numeric_limits<int64_t>::max() == (1UL << 63) - 1);
-- 
2.34.1

