From: heoh <h0h6h2h5@gmail.com>
Subject: [PATCH] WASI: Disable timezone-related features in Bazel
 configuration

---
 .bazelrc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.bazelrc b/.bazelrc
index 65511dc6..efd3c0bc 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -87,6 +87,10 @@ build:wasi --copt=-D_WASI_EMULATED_SIGNAL
 # Disable unsupported features
 build:wasi --copt=-DGTEST_HAS_STREAM_REDIRECTION=0
 build:wasi --copt=-DOPENSSL_NO_SOCK
+build:wasi --copt=-DU_HAVE_TZSET=0
+build:wasi --copt=-DU_HAVE_TZNAME=0
+build:wasi --copt=-DU_HAVE_TIMEZONE=0
+build:wasi --copt=-DU_HAVE_TM_GMTOFF=0
 
 # Linker flags
 build:wasi --linkopt=--target=wasm32-wasip1-threads
-- 
2.34.1

