From: heoh <h0h6h2h5@gmail.com>
Subject: [PATCH] WASI: Add workaround for asm.js compatibility in Bazel
 configuration

---
 .bazelrc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.bazelrc b/.bazelrc
index e4a6414f..9a5c8dd5 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -92,6 +92,9 @@ build:wasi --copt=-DU_HAVE_TZNAME=0
 build:wasi --copt=-DU_HAVE_TIMEZONE=0
 build:wasi --copt=-DU_HAVE_TM_GMTOFF=0
 
+# Workarounds
+build:wasi --copt=-D__asmjs__=1
+
 # Linker flags
 build:wasi --linkopt=--target=wasm32-wasip1-threads
 build:wasi --linkopt=-stdlib=libc++
-- 
2.34.1

